crates/subspace-proof-of-space/Cargo.toml

```diff
@@ -16,11 +16,29 @@ include = [
 bench = false
 
 [dependencies]
+bitvec = { version = "1.0.1", default-features = false, features = ["alloc", "atomic"], optional = true }
+blake3 = { version = "1.3.3", default-features = false, optional = true }
+chacha20 = { version = "0.9.1", default-features = false, optional = true }
 subspace-chiapos = { git = "https://github.com/subspace/chiapos", rev = "3b1ab3ca24764d25da30e0c8243e0bf304b776a5", optional = true }
 subspace-core-primitives = { version = "0.1.0", path = "../subspace-core-primitives", default-features = false }
 
+# Ugly workaround for https://github.com/rust-lang/cargo/issues/1197
+[target.'cfg(any(target_os = "linux", target_os = "macos", all(target_os = "windows", target_env = "gnu")))'.dependencies.sha2]
```

Comment:
If sha2 is an issue we can replace it with blake(2 or 3) later. It's only used for qualities anyway.

Reply:
It is not an issue, it is just assembly implementation of Sha2 that is not working with MSVC on Windows (at least for now), only with GNU compiler.